src/Syntax.jl

```diff
@@ -988,4 +988,81 @@ end
 
 
 
+"""
+stock_and_flow_simpler(;
+    stocks::Vector{Symbol}
+    flows::Vector{Pair{Symbol, Pair{Expr, Symbol}}}
+    dynamic_variables::Vector{Pair{Symbol, Expr}}
+    parameters::Vector{Symbol}
+    sums::Vector{Pair{Symbol, Vector{Symbol}}}
+    )
+    
+Uses StockAndFlowBlock to create a StockFlowF, but allows for a simpler syntax.
+Useful for creating StockFlows dynamically.
+```julia
+stock_and_flow_simpler(
```

Comment:
Would another macro like
``` julia
quote
  stocks = A, B, C
  flows  = A => f(v1) => CLOUD,
           B => f2(v1 + A) => C
  sums   = N => [A, C],
           NI => NONE
  dynamic_variables = v1 => A * B
end
```
help the situation at all? This does parse in a quote block.

Reply:
Idea was this would help with automating dynamic rewriting.  Could say what indices you wanted to associate, and it would try creating a new StockFlow with that and defining a homomorphism to it.  I figured I wouldn't want to use a macro, as those are generally done before the code actually runs.

It's fine if this doesn't end up getting merged, I just need to find a way to do rewritings more easily (eg, with pertussis having a ton of duplicate deltas, the infection variable being recalculated, etc.)